Applications/Resnet/PyTorch/main.py

```diff
@@ -90,6 +107,17 @@ def forward(self, x):
         out = self.linear(out)
         return out
 
+    def save_bin(self, name):
```

Comment:
This could be in transLayer_v2.py.

Reply:
> This could be placed in transLayer_v2.py.

That's a good idea. I'll move it to transLayer_v2.py. thanks!